acquire/acquire.py

```diff
@@ -2036,29 +2038,30 @@ class VolatileProfile:
 }
 
 
+def exit_success(default_args: list[str]):
```

Comment:
The GUI already has support for a progress bar, why not integrate the exit screens with the GUI? I.e. make a text-based GUI?

Reply:
No, this is logging related.